AElf.Cryptography/AElfKeyStore.cs

```diff
@@ -84,13 +85,11 @@ public Errors OpenAsync(string address, string password, bool withTimeout = true
             return Errors.None;
         }
 
-        private void RemoveAccount(object accObj)
```

Comment:
It is very strange to have methods named `OpenAccount`/`RemoveAccount`. Change it to `CloseAccount`